ts/Maps/MapView.ts

```diff
@@ -926,16 +926,35 @@ class MapView {
                     }
 
 
-                } else {
-
-                    const scale = this.getScale();
-
-                    const newCenter = this.projection.inverse([
-                        mouseDownCenterProjected[0] +
-                            (mouseDownX - chartX) / scale,
-                        mouseDownCenterProjected[1] -
-                            (mouseDownY - chartY) / scale
-                    ]);
+                } else { // #17238
+                    const scale = this.getScale(),
+                        flipFactor = this.projection.hasCoordinates ? 1 : -1,
+                        options = chart.userOptions;
+
+                    let newCenter;
+
+                    if (options.chart && options.chart.panning &&
+                            options.chart.panning.type === 'x') {
+                        newCenter = this.projection.inverse([
+                            mouseDownCenterProjected[0] +
+                                (mouseDownX - chartX) / scale,
+                            mouseDownCenterProjected[1]
+                        ]);
+                    } else if (options.chart && options.chart.panning &&
+                                    options.chart.panning.type === 'y') {
+                        newCenter = this.projection.inverse([
+                            mouseDownCenterProjected[0],
+                            mouseDownCenterProjected[1] -
+                                (mouseDownY - chartY) / scale * flipFactor
+                        ]);
+                    } else {
```

Comment:
Do we need to support individual `x` and `y` panning for a map? The OP is only reporting the `flipFactor` issue...

Reply:
While testing it, I have noticed that the `chart.panning.type` no longer works, so I've decided to also fix it here. But it's your call whether we should just fix the `flipFactor` issue, or leave the  `x` and `y` `panning.type` aswell?

Reply:
@kamil-musialowski Thanks! I don't think we should support `x` and `y` panning in Maps. I don't think it makes much sense on a geo map. Unless people are using it, we risk adding unnecessary bloat.

Reply:
You are right. When I think about it now, it doesn't make much sense now to limit the panning in maps. In that case, should we remove it from [Highmaps API](https://api.highcharts.com/highmaps/chart.panning.type)?

Reply:
Yes we should. I assume that description is just inherited from vanilla charts.

Reply:
@TorsteinHonsi Done and ready for merge!